softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java

```diff
@@ -6,15 +6,22 @@
 import com.softgroup.common.protocol.Response;
 import com.softgroup.common.protocol.ResponseData;
 
-public class AbstractRequestHandler<T extends RequestData, R extends ResponseData> implements RequestHandler {
-	@Override
-	public String getName() {
-		return null;
-	}
+public abstract class AbstractRequestHandler<T extends RequestData, R extends ResponseData>
+		implements RequestHandler {
+//	@Override
+//	public String getName() {
+//		return null;
+//	}
 
 	@Override
 	public Response<R> handle(Request<?> msg) {
-		return null;
+		Request<T> requestObj = new Request<T>();
+		requestObj.setHeader(msg.getHeader());
+
+		//requestObj is created after Jackson mapper conversion
+		return doHandle(requestObj);
 	}
 
+	public abstract Response<R> doHandle(Request<?> requestObj);
```

Comment:
@Dominys 
handle() parses object and after that doHandle() receives it. I suggest, it includes repeatable actions, when doHandle() will be implemented by other handlers individually